packages/client/src/components/MaximizableView/MaximizableView.tsx

```diff
@@ -0,0 +1,61 @@
+import { FC, useRef } from 'react';
+
+import { useFullscreenStatus } from '@utils/useFullscreenStatus';
+
+import { TMaximizableViewProps } from './types';
+
+export const MaximizableView: FC<TMaximizableViewProps> = ({ children, backgroundColor }) => {
+  const maximizableElement = useRef(null);
+
+  let isFullscreen, setIsFullscreen;
+  let errorMessage;
+  try {
+    const fullscreen = useFullscreenStatus(maximizableElement);
+    setIsFullscreen = fullscreen.setFullscreen;
+    isFullscreen = fullscreen.isFullscreen;
+  } catch (e) {
+    errorMessage = 'Полноэкранный режим не поддерживается';
+    isFullscreen = false;
+    setIsFullscreen = undefined;
+  }
+
+  const handleExitFullscreen = () => document.exitFullscreen();
+
+  const fullscreenButton = isFullscreen ? (
+    <button
+      className="btn btn-primary my-6 text-2xl absolute right-12"
+      onClick={handleExitFullscreen}
```

Comment:
Создание `handleExitFullscreen` нужно
```suggestion
      onClick={document.exitFullscreen}
```

Reply:
Так будет ошибка: 
`Uncaught (in promise) TypeError: Failed to execute 'exitFullscreen' on 'Document': Illegal invocation
    at HTMLUnknownElement.callCallback2 (react-dom.development.js:4164:14)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:4213:16)
    at invokeGuardedCallback (react-dom.development.js:4277:31)
    at invokeGuardedCallbackAndCatchFirstError (react-dom.development.js:4291:25)
    at executeDispatch (react-dom.development.js:9041:3)
    at processDispatchQueueItemsInOrder (react-dom.development.js:9073:7)
    at processDispatchQueue (react-dom.development.js:9086:5)
    at dispatchEventsForPlugins (react-dom.development.js:9097:3)
    at react-dom.development.js:9288:12
    at batchedUpdates$1 (react-dom.development.js:26140:12)
c`
Можно написать так `onClick={() =>document.exitFullscreen()}`